.github/workflows/mark-issue-duplicate.yml

```diff
@@ -8,8 +8,9 @@ on:
 jobs:
   mark-duplicate:
     runs-on: ubuntu-latest
+    if: ${{ !github.event.issue.pull_request && startsWith(github.event.comment.body, "Duplicate of ") }}
```

Comment:
A note: The syntax only allow use single quote, fixed in https://github.com/prettier/prettier/commit/a5118601e5aa1e9a1e2fe0b5bfcc5ad4ea79e822